lti_consumer/lti_1p3/consumer.py

```diff
@@ -475,16 +475,22 @@ def lti_ags(self):
     def enable_ags(
         self,
         lineitems_url,
+        lineitem_url,
```

Comment:
This should be an optional parameter.

Reply:
Both ``lineitems_url`` and ``lineitem_url`` are now optional.